scss/misc.scss

```diff
@@ -44,8 +44,44 @@
   padding: 3px;
 }
 
-//alas too broad, is overruling links that we made grey because they are minor, for exmaple, date link on a post, dont have alternative yet
-// a,
-// a:visited {
-//   color: var(--accent-color);
-// }
+// alas too broad,
+// is overruling links that we made grey because they are minor,
+// for exmaple,
+// date link on a post,
+// dont have alternative yet
+a,
```

Comment:
This is going to cause the same issue as I commented.

Reply:
I tested it but it didn't seem to change the link in question. Can you pull locally?